common/persistence/cassandra/cassandraMetadataPersistenceV2.go

```diff
@@ -573,3 +600,15 @@ func (m *cassandraMetadataPersistenceV2) deleteDomain(name, ID string) error {
 
 	return nil
 }
+
+func (m *cassandraMetadataPersistenceV2) getFailoverConfig(
+	request *p.InternalUpdateDomainRequest,
+) p.DomainFailoverConfig {
+
+	config := p.DomainFailoverConfig{}
+	if request.FailoverConfig != nil {
+		config.StartTime = request.FailoverConfig.StartTime
+		config.Timeout = request.FailoverConfig.Timeout
```

Comment:
What was the idea of having a timeout here instead of just specifying an end time? By giving a start time and a timeout I am wondering if the application logic is going to have to do startTime + timeout >= something in a bunch a places?

Reply:
You are right. I will use end time

Reply:
Updated